docs/source/optimizations/qscout/qscout_css.ipynb

```diff
@@ -30,11 +32,14 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "id": "8c861954",
    "metadata": {},
    "source": [
-    "## Imports and Token"
+    "## Imports and API Token\n",
+    "\n",
+    "For this tutorial, we will just need the imports listed below to run the Superstaq optimizations. If you do not have `cirq-superstaq` installed on your machine, you can follow the guide [here](https://docs-superstaq.readthedocs.io/en/latest/get_started/installation.html#installing-cirq-superstaq) to set it up and its dependencies. "
```

Comment:
imo we should lead with something like "this notebook uses cirq-superstaq, our superstaq client for Cirq; you can try it out by running pip install cirq-superstaq" instead of making it just one of the "imports listed below"

Reply:
I initially assumed that users would go to the installation guide first, but I've added your suggestion to be more self-contained.